packages/load-test/src/cli.ts

```diff
@@ -0,0 +1,269 @@
+#!/usr/bin/env tsx
```

Comment:
do we want a `build` script and use node instead of tsx? maybe it's overkill at this point since it's dev-mode flavored CLI